client/post-editor/editor-drawer/index.jsx

```diff
@@ -245,8 +246,11 @@ const EditorDrawer = React.createClass( {
 		}
 
 		const { plan } = this.props.site;
+		const { blog_public } = this.props.siteSettings;
+		const isSitePrivate = parseInt( blog_public, 10 ) !== 1;
```

Comment:
@mcsf From what I can see, we always expect `blog_public` to be a number ([either -1, 0, or 1](https://github.com/Automattic/wp-calypso/blob/b73f49af4f72f859f004d16d6274f76c6d1168da/client/my-sites/site-settings/form-general.jsx#L181)). I suspect the `parseInt()` is just there to be extra sure. cc @roundhill maybe you could confirm? I borrowed this logic from your work on SEO settings [here](https://github.com/Automattic/wp-calypso/pull/5365#discussion-diff-63278075).